src/components/MaterialUI/Navigation/DropDownMenu.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, isValidElement, cloneElement } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import Text from "../../Text";
+import DropDownMenuProps from "./DropDownMenuProps";
+
+const useStyles = makeStyles(theme => ({
+	menuItem: {
+		fontFamily: theme.typography.fontFamily,
+		textTransform: "none",
+		color: theme.palette.text.primary,
+		borderRadius: 0,
+		"&:hover, &:focus": {
+			borderRadius: 0,
+			boxShadow: "none",
+			backgroundColor: theme.palette.primary.light,
+		},
+	},
+	menu: {
+		border: `${theme.spacing(0.1)} solid ${theme.palette.grey.borders}`,
+		boxShadow: `${theme.spacing(0, 0.5, 0.7, 0)} ${theme.palette.grey.borders}`,
+		borderRadius: theme.spacing(0.3),
+	},
+}));
+
+const DropDownMenu = ({ payload, menuItems, children, dropDownMenuProps = new DropDownMenuProps() }) => {
```

Comment:
I have a concern about creating new prop instance for default value of props. I just don't understand what for.. And another thing you missed is actually to check the passed props parameter if it's correct. You've created isDropDownMenuProps  but, didn't use here for some reason. Again the pattern which is used to check it could be found in other wrappers. Also don't forget to add extra unit test for it (samples of such unit tests presents for other wrappers as well).